gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageGrpcIntegrationTest.java

```diff
@@ -0,0 +1,206 @@
+package com.google.cloud.hadoop.gcsio.integration;
+
+import static com.google.cloud.hadoop.gcsio.integration.GoogleCloudStorageTestHelper.assertObjectContent;
+import static com.google.cloud.hadoop.gcsio.integration.GoogleCloudStorageTestHelper.writeObject;
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+import static org.junit.Assert.assertThrows;
+
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorage;
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorageImpl;
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorageItemInfo;
+import com.google.cloud.hadoop.gcsio.ListObjectOptions;
+import com.google.cloud.hadoop.gcsio.StorageResourceId;
+import com.google.common.collect.Lists;
+import com.google.common.flogger.GoogleLogger;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.ClosedChannelException;
+import java.nio.channels.SeekableByteChannel;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class GoogleCloudStorageGrpcIntegrationTest {
+
+  // Prefix this name with the prefix used in other gcs io integrate tests once it's whitelisted by
+  // GCS to access gRPC API.
+  private static final String BUCKET_NAME = "gcs-grpc-team-dataproc-it";
+
+  private static GoogleCloudStorage createGoogleCloudStorage() {
+    try {
+      return new GoogleCloudStorageImpl(
+          GoogleCloudStorageTestHelper.getStandardOptionBuilder().setGrpcEnabled(true).build(),
+          GoogleCloudStorageTestHelper.getCredential());
+    } catch (IOException e) {
+      GoogleLogger logger = GoogleLogger.forEnclosingClass();
```

Comment:
I think it's better to add `throws IOException` to the method signature and delete this try-catch block.

Reply:
Done.